app/models/mets_file_display_format.rb

```diff
@@ -0,0 +1,24 @@
+class METSFileDisplayFormat
+
+  class << self
+    attr_accessor :display_format_translations
+  end
+
+  def self.get(mets_file)
+    return if mets_file.root_type_attr.blank?
+    load_display_format_translations unless display_format_translations
+    type_attr = mets_file.root_type_attr.split(':').last.downcase
+    display_format_translations[type_attr] || type_attr
+  end
+
+  def self.load_display_format_translations
```

Comment:
Why a separate translation file?  If we need one, can we put it in `config/locales` so that it gets automatically loaded?